image/autodetect.go

```diff
@@ -61,8 +61,6 @@ func Autodetect(path string) (string, error) {
 
 	switch mimeType {
 	case "application/x-gzip":
-		return TypeImage, nil
-
 	case "application/octet-stream":
```

Comment:
`case "application/x-gzip", "application/octet-stream":` would be fine, but what you have here is:

```go
case "application/x-gzip":
case "application/octet-stream":
```

And testing a similar layout locally:

```
$ cat x.go 
package main
import ("fmt")
func main() {
        var a = 0
        switch a {
        case 0:
        case 1:
                fmt.Println("hi")
        }
}
$ go run x.go 
```

I don't see `hi` in my stdout.